packages/backend/src/apps/todoist/auth/index.ts

```diff
@@ -12,10 +12,22 @@ export default {
       readOnly: false,
       value: null,
       placeholder: null,
-      description:
-        'Name your connection (only used for Automatisch UI).',
+      description: 'Name your connection (only used for Automatisch UI).',
       clickToCopy: false,
     },
+    {
```

Comment:
@katrieltsepelevish, could you please display this field as the first for consistency with other integrations?

Reply:
Done